python/xorbits/_mars/deploy/oscar/supervisor.py

```diff
@@ -52,6 +52,21 @@ def parse_args(self, parser, argv, environ=None):
 
         args.supervisors = f"{args.supervisors},{args.endpoint}".strip(",")
 
+        oscar_config = self.config.get("oscar")
+        external_addr_scheme = None
+        if oscar_config is not None:
+            numa_addr_scheme = oscar_config.get("numa").get("external_addr_scheme")
+            gpu_addr_scheme = oscar_config.get("gpu").get("external_addr_scheme")
+            external_addr_scheme = numa_addr_scheme or gpu_addr_scheme
+        if external_addr_scheme:  # pragma: no cover
```

Comment:
I think we can also do these to the ``workers``. Since we can get ``oscar_config`` here, then add the ``ucx://`` prefix to the ``-s`` parameter of the cmdline according to the ``numa_addr_scheme``. In this way, the user can leave the startup command unchanged.
``python -m xorbits.worker -s <supervisor_ip>:<supervisor_port>``

Reply:
I updated worker.py. Note that users should use `-f` option to pass the config file. Otherwise, it may not work as expected.

Reply:
> I updated worker.py. Note that users should use `-f` option to pass the config file. Otherwise, it may not work as expected.

Thanks. Currently the code change in ``supervisor.py`` and ``worker.py`` is exactly the same, could you please turn it into a something like ``util`` method to reduce code duplication?